src/server_state.cpp

```diff
@@ -811,6 +822,21 @@ void wsrep::server_state::last_committed_gtid(const wsrep::gtid& gtid)
            last_committed_gtid_.seqno() + 1 == gtid.seqno());
     last_committed_gtid_ = gtid;
     cond_.notify_all();
+
```

Comment:
This whole method seems to be dead code, please consider applying https://github.com/codership/wsrep-lib/commit/a921a4f57973948db68e4bae9eee7e77a2ff0771 and corresponding cleanup for MySQL-wsrep/MariaDB: https://github.com/codership/mariadb-server/pull/244/commits/448a49fb776ab6a1727a330409ff4da319e2a590.